src/App.js

```diff
@@ -1,23 +1,45 @@
-import logo from './logo.svg';
 import './App.css';
+import React, {useState} from 'react';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFoodForm from './components/AddFoodForm';
+import Search from './components/Search';
+
 
 function App() {
+  const [dishes, setDishes] = useState(foods)
+  const [showDishes, setShowDishes] = useState(foods);
+
+  const createFood = (newFood) => {
+    const updatedFood = [newFood, ...dishes]
+    setDishes(updatedFood)
+  }
+
+  const searchFood = (searchQuery) => {
+    const filteredFood = dishes.filter((food) => 
+    food.name.toLowerCase().includes(searchQuery.toLowerCase())
+    );
+    setDishes(filteredFood);
+  }
+
+  const deleteFood = (name) => {
+    const filteredFood = dishes.filter((food) => food.name !== name);
+    setDishes(filteredFood);
+    setShowDishes(filteredFood);
+  };
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className='container'>
+      <Search filterFood={searchFood}/>
+      <AddFoodForm createFood = {createFood} className='crtForm'/>
+      <h1 className='tlt'>Food List</h1>
+      <div className="App">
+        {dishes.map((dish) => {
```

Comment:
Now that you have a new 'useState' the is the result of the 'Search' you need to loop over it, so instead of:
` {dishes.map((dish) => {`
Do:
`  {showDishes.map((dish) => {`